lms/djangoapps/onboarding/views.py

```diff
@@ -41,6 +42,19 @@
 log = logging.getLogger("edx.onboarding")
 
 
+def check_unattended_surveys(user_extended_profile):
```

Comment:
There is already a method in UserExtendedProfile `unattended_surveys` which is being used inside this method? can renamed this in a way like it serve the purpose what additional we are doing in it?

Reply:
Please suggest a name, I have spent sometime and came up with this name